src/app/components/question-view/question-view.component.html

```diff
@@ -3,19 +3,26 @@
   <div class="card-header" *ngIf="questions.length > 0">
     <div *ngIf="question.tags?.length > 0" class="row">
       <div *ngFor="let tag of question.tags; let i = index">
-        <label class="badge badge-{{tagStyles[i]}}">{{question.tags[i]}}</label>
+        <label class="badge badge-{{tagStyles[i]}}">{{question.tags[i].toString().split('_').join(' ')}}</label>
       </div>
     </div>
   </div>
   <div class="card-body">
-    <pre class="text-muted"> <markdown ngPreserveWhitespaces>{{question.content}}</markdown></pre>
+    <pre class="text-muted"> <markdown>{{question.content}}</markdown></pre>
     <button class="btn btn-secondary" *ngIf="!question.revealAnswer" (click)="toggleAnswer(question, true)">Show
       answer</button>
     <pre class="text-muted" *ngIf="question.revealAnswer">
         <li class="list-group-item">
           <button class ="btn btn-secondary" (click)="toggleAnswer(question, false)">Hide answer</button>
-          <markdown ngPreserveWhitespaces>{{question.answer}}</markdown>
+          <markdown>{{question.answer}}</markdown>
         </li>
       </pre>
   </div>
+</div>
+<div *ngIf="questions.length === 0">
+  <div class="card">
+    <div class="card-body text-middle">
+      <h2>Couldn't find anything with selected tags</h2>
+    </div>
+  </div>
```

Comment:
When page loads its shown too, while that shouldn't be the case. 
![image](https://user-images.githubusercontent.com/24475158/79095966-23d75800-7d5c-11ea-8536-2512dbec0cd0.png)

Also, its a bit too big communicate :D

Reply:
Fixed:
- smaller message
- showing only when there's no questions (not while loading)
- loading animation is enough in my opinion